unpin widgettastic/navmazing

pinning navmazing/widgettastic to very exact versions makes airgun incompatible with the rest of the ecosystem as it always requests that specific version

please use less specific pins or at last minimal version pins

I finally have results from Satellite QE automation run when Airgun dependencies are not pinned.

1. Logging changes in navmazing 1.1.6 prevents Airgun from starting up with following error:
```
../../shiningpanda/jobs/ad0ef6b5/virtualenvs/d41d8cd9/lib64/python3.6/copy.py:150: in deepcopy
    y = copier(x, memo)
../../shiningpanda/jobs/ad0ef6b5/virtualenvs/d41d8cd9/lib64/python3.6/copy.py:240: in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
                    else:
                        reductor = getattr(x, "__reduce_ex__", None)
                        if reductor:
>                           rv = reductor(4)
E                           TypeError: can't pickle _thread.RLock objects
```
Root cause is line: https://github.com/SatelliteQE/airgun/blob/master/airgun/session.py#L185

2. With above issue fixed (by forcing navmazing 1.1.5), I got 63 tests failed, as opposed to 62 failed in last normal automation run (both out of 420). However, 33 failed in unpinned run did pass in usual pinned run. I believe these differences has more to do with stability of our automation and intermittent infrastructure problems. Nevertheless, some are probably genuine regressions that will require additional work on our side.

--- a/setup.py
+++ b/setup.py
@@ -16,11 +16,11 @@ setup(
     install_requires=[
         'cached_property',
         'fauxfactory',
-        'navmazing==1.1.4',
+        'navmazing',
         'pytest',
         'wait_for',
-        'widgetastic.core==0.33',
-        'widgetastic.patternfly==0.0.38'
+        'widgetastic.core',
+        'widgetastic.patternfly'
     ],
     packages=find_packages(exclude=['tests*']),
     package_data={'': ['LICENSE']},

